packages/dev/core/src/Maths/math.scalar.functions.ts

```diff
@@ -18,6 +18,18 @@ export function WithinEpsilon(a: number, b: number, epsilon: number = 1.401298e-
     return Math.abs(a - b) <= epsilon;
 }
 
+/**
+ * Boolean : true if the number is outside a range
+ * @param num number
+ * @param min min value
+ * @param max max value
+ * @param epsilon (default = Number.EPSILON)
+ * @returns true if the number is between min and max values
+ */
+export function OutsideRange(num: number, min: number, max: number, epsilon: number = 1.401298e-45): boolean {

```

Comment:
Why not using `Number.EPSILON` instead of `1.401298e-45`? Also, Chromes give me `2.220446049250313e-16` for `Number.EPSILON`:
![image](https://github.com/user-attachments/assets/eb235af4-0c6f-4f7f-a38e-1c95188ea3db)

[...]
Looking at existing code above, I can see that `1.401298e-45` is the default epsilon value we use elsewhere, so I think you should simply remove the reference to `Number.EPSILON` from the function doc.